src/ui/units/wizard/components/Dialogs/DialogPlaceholder/DialogPlaceholder.tsx

```diff
@@ -786,20 +789,38 @@ class DialogPlaceholder extends React.PureComponent<Props, State> {
             },
         };
 
-        if (
-            axisType === SETTINGS.TYPE.LOGARITHMIC &&
-            settings.scaleValue === SETTINGS.SCALE_VALUE.ZERO_MAX
-        ) {
-            updatedState.settings = {
-                ...updatedState.settings,
-                scaleValue: SETTINGS.SCALE_VALUE.MIN_MAX,
-            };
-            updatedState.tooltipType = 'scaleValue';
-            updatedState.tooltipContent = (
-                <span data-qa={DialogPlaceholderQa.TooltipZeroToMaxScale}>
-                    {i18n('wizard', 'tooltip_zero-to-max-scale')}
-                </span>
-            );
+        updatedState.tooltipType = undefined;
+        updatedState.tooltipContent = '';
+
+        if (axisType === SETTINGS.TYPE.LOGARITHMIC) {
+            if (settings.scaleValue === SETTINGS.SCALE_VALUE.ZERO_MAX) {
+                updatedState.settings = {
+                    ...updatedState.settings,
+                    scaleValue: SETTINGS.SCALE_VALUE.MIN_MAX,
+                };
+                updatedState.tooltipType = 'scaleValue';
+                updatedState.tooltipContent = (
+                    <span data-qa={DialogPlaceholderQa.TooltipZeroToMaxScale}>
+                        {i18n('wizard', 'tooltip_zero-to-max-scale')}
+                    </span>
+                );
+            } else if (
+                settings.scale === SETTINGS.SCALE.MANUAL &&
+                (Number(settings?.scaleValue?.[0]) <= 0 || Number(settings?.scaleValue?.[1]) <= 0)
+            ) {
+                updatedState.settings = {
+                    ...updatedState.settings,
+                    scale: SETTINGS.SCALE.AUTO,
+                    scaleValue: SETTINGS.SCALE_VALUE.MIN_MAX,
+                };
+
+                updatedState.tooltipType = 'scaleValue';
+                updatedState.tooltipContent = (
+                    <span data-qa={DialogPlaceholderQa.TooltipZeroToMaxScale}>
+                        {i18n('wizard', 'tooltip_zero-to-max-scale')}
```

Comment:
It would be great to display a tooltip with a specific min value.
Now it is not very clear why it will be said about zero with a negative value